cli/src/main/kotlin/io/spine/protodata/cli/Main.kt

```diff
@@ -356,16 +340,38 @@ internal class Run(version: String) : CliktCommand(
         try {
             return createByName(className, classLoader)
         } catch (e: ClassNotFoundException) {
-            error(e.message)
-            error("Please add the required class `$className` to the user classpath.")
+            printError(e.message)
```

Comment:
In this throwable a stacktrace is also very valuable. Can we please print it as well?